src/components/AcceptDialog.tsx

```diff
@@ -60,7 +85,10 @@ const AcceptDialog: FunctionComponent<Props> = (props) => {
     values: DocumentSubmissionUpdateForm,
     userUpdatedBy: string
   ) => {
-    if (values.validUntilDates && values.validUntilDates.length > 0) {
+    if (
+      values.validUntilDates &&
+      !values.validUntilDates.every((x: string) => x == '' || undefined)
```

Comment:
Empty string or undefined, to handle the case when a user does not add anything (undefined) or enters a date then removes it (empty string)